pkg/cron/cron.go

```diff
@@ -96,7 +96,8 @@ func notifyOnErr(err error, backoffDelay time.Duration) {
 // RequestCall - ZMS call for update crons
 func (c *Cron) requestCall() error {
 	master := false
-	domains, etag, err := c.zmsClient.GetSignedDomains("", "true", "", &master, c.etag)
+	conditions := false
+	domains, etag, err := c.zmsClient.GetSignedDomains("", "true", "", &master,&conditions, c.etag)
```

Comment:
Ditto. (one space after comma)
Also, can we make sure what's the effect of this new parameter? Why it has to be false by default? (why not true?)

Reply:
Hi @zetaj , @bhaskaramVaranasi ,
The `conditions` variable has been added in  v1.10.22 ref: https://github.com/AthenZ/athenz/releases/tag/v1.10.22
Here is the [pr-link](https://github.com/AthenZ/athenz/pull/1523/files) for it.
Here is the [explanation](https://github.com/AthenZ/athenz/pull/1523/files#diff-5f38b5a48f66920c30ca4948aeda321295fa72bcd2e4f53f71c30feb0fc9a42eR2024-R2026) for the conditions variable from the pr.
Since `conditions is a boolean`, It Controls whether assertion conditions are included in the response from the ZMS server.
When set to `false` (default)- The response will not include assertion conditions
When set to `true`, the ZMS server will include any configured assertion conditions within the response assertions.